friendlist/src/main/resources/db/migration/V1__PHPM_TABLE.sql

```diff
@@ -0,0 +1,35 @@
+-- public.friend_request definition
```

Comment:
Can we add a query to if schema exists or not?
CREATE SCHEMA [IF NOT EXISTS] schema_name;